lib/frrscript.h

```diff
@@ -24,13 +24,51 @@
 #ifdef HAVE_SCRIPTING
 
 #include <lua.h>
+#include <nexthop.h>
+#include <nexthop_group.h>
 #include "frrlua.h"
 #include "bgpd/bgp_script.h" // for peer and attr encoders/decoders
 
+/* Forward declarations */
+extern struct zebra_dplane_ctx ctx;
+extern void lua_pushzebra_dplane_ctx(lua_State *L,
+				     const struct zebra_dplane_ctx *ctx);
+extern void lua_decode_zebra_dplane_ctx(lua_State *L, int idx,
+					struct zebra_dplane_ctx *ctx);
```

Comment:
These should go in the `extern "C"` block

Reply:
updated in https://github.com/FRRouting/frr/pull/9440/commits/730d6378ff9ec8cfca23f117c3d91310f14be3df